reo/tests/posts/test_cold_POST.json

```diff
@@ -0,0 +1,1697 @@
+{
```

Comment:
@Bill-Becker if you remove `test_cold` please also remove this file

Reply:
Keeping test_cold.